eng/scripts/Update-PkgVersion.ps1

```diff
@@ -6,11 +6,11 @@ Bumps up package versions after release
 The root of the repo (defaults to ${PSScriptRoot}/..)
 
 .PARAMETER ServiceDirectory
-The service directory under <repo-root>/sdk/ used to find version.hpp
+The service directory under <repo-root>/sdk/ used to find package_version.hpp
```

Comment:
I didn't find any string for the expected path like `inc/azure/core` or something. 
Does it means that the script will try to find `package_version.hpp` anywhere under the `sdk/ServiceDirectory` path?

Reply:
See https://github.com/Azure/azure-sdk-for-cpp/pull/2166/files#diff-f78d4a430718b3caaf9acaafbcc127358ac1be36f198386ebefe14891c18870eR7 it looks under the `sdk/$ServiceDirectory/$PackageName` directory for the file.